formula-android/src/main/java/com/instacart/formula/fragment/FormulaFragment.kt

```diff
@@ -113,7 +113,7 @@ class FormulaFragment<RenderModel> : Fragment(), BaseFormulaFragment<RenderModel
         return stateRelay.value
     }
 
-    override fun getFragmentContract(): FragmentContract<RenderModel> {
+    override fun getFragmentKey(): FragmentContract<RenderModel> {
```

Comment:
Is this the right return type?

Reply:
Good call, will update to `FragmentKey`.